.github/workflows/case-data-update-dev.yml

```diff
@@ -0,0 +1,40 @@
+name: Update case data in dev
+
+on:
+  push:
+    # Update the data when the schema changes.
+    branches: [master]
+    paths: ["data-serving/data-service/schemas/**"]
+
+jobs:
+  # NOTE: This job should bee kept in sync with the prod version.
+  update-case-data-dev:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install MongoDB CLIs
+        run: |
+          wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | sudo apt-key add -
+          echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.2.list
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 5DC22404A6F9F1CA 656408E390CFB1F5
```

Comment:
👀 
/me what is this madness

Reply:
NO QUESTIONS

ok fine, it was necessary because the script was having trouble looking up the keyserver for mongodb packages... it was weird... it might have just been a local problem but i'm not taking any chances